book/index.adoc

```diff
@@ -18,6 +18,9 @@
 :description: HTML meta tag content for description
 :keywords: HTML meta tag content for keywords
 :icons: font
+:pdf-themesdir: {docdir}/themes
```

Comment:
@ksdhans one more question - where does `docdir` come from here?

Reply:
It's one of the built-in values. Yes, it's relative to the maindoc. I had to put it in there or it wouldn't work properly for some reason. No idea why ./ wasn't enough, as it should point to the same place.

Reply:
Thanks!